cmd/documentation/cmd_templates/apichecker.tmpl

```diff
@@ -0,0 +1,92 @@
+{{define "cmd apichecker" -}}
+{{template "header" .}}
+## Current Features for {{.Name}}
+
++ Checks for API updates
++ Can automatically update Trello checklist for the updates required
++ Supports trello integration
+
+#### This tool tracks changes in exchange API documentation
+#### Keeps track of all the updates using the GoCryptoTrader trello board
+
+Be aware, this tool will:
+- Automatically update the live trello board if API keys and trello information are provided.
+- Automatically update the main json updates file, however a backup of the copy before the updates will be stored.
+
+## Usage
+
++ To run a real check for updates, parse Trello API info as flags or add them to the updates.json file and use the following command from apichecker folder in GCT:
+
+###### Linux/OSX
+GoCryptoTrader is built using [Go Modules](https://github.com/golang/go/wiki/Modules) and requires Go 1.11 or above
+Using Go Modules you now clone this repository **outside** your GOPATH
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader/cmd/apichecker
+go build
+./apichecker
+```
+
+###### Windows
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader\cmd\apichecker
+go build && apichecker.exe
+```
+
++ Upon addition of a new exchange, to update Trello checklist and to add the exchange to updates.json the following would need to be done:
```

Comment:
Could you please also add a bit to the readme about your latest feature to populate everything automatically. eg
`apichecker.exe -create -apikey=apiKey -apitoken=apiToken -boardname=blah -firstrun`